### PR TITLE
Adapt

### PR DESCRIPTION
**Pull request by veronaten**
I adapted my project for the tablet and mobile versions.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -127,7 +127,17 @@ <h1 class="title title_fz48 promo__title">Я web-разработчик из г
     <section class="about">
         <div class="container">
             <div class="about__wrapper">
-                <div class="about__photo"><img src="img/main_photo.jpg" alt="photo">
+                <div class="about__photo">
+                    <!-- <img srcset="img/main_photo_min.jpg 300w,
+                                img/main_photo.jpg 400w"
+                        sizes="(max-width: 576px) 280px,
+                                (min-width: 577px) 400px"
+                    src="img/main_photo.jpg" alt="photo"> -->
```
Comment (magicsd) [inline src/index.html]: Remove unused code

File: src/index.html [hunk 2]
```diff
@@ -447,7 +457,7 @@ <h2 class="title title_fz16 contacts__title">Контакты</h2>
                     <div class="title title_fz14 contacts__text">Любым удобным для вас способом:</div>
                     <div class="contacts__social">
                         <a href="#" class="contacts__link">
-                            <img src="icons/social/facebook.svg" alt="Facebook">
+                            <img src="icons/social/Facebook turn.svg" alt="Facebook">
```
Comment (magicsd) [inline src/index.html]: Filename must be in camelCase, snake_case or dash-case.
E.g. `facebook-turn.svg`

Comment (veronaten) [reply src/index.html]: ok, understand)

Comment (magicsd) [reply src/index.html]: Just a reminder: `Facebook-turn.svg` and `facebook-turn.svg` are two diffetent files

**Pull request merged**